src/main/java/jalau/at18/katas/bankocr/sergio/OpticCharacterReader.java

```diff
@@ -0,0 +1,79 @@
+package jalau.at18.katas.bankocr.sergio;
+
+import java.util.Arrays;
+
+public class OpticCharacterReader {
+    private Numbers numbers;
+
+    public int numberOne(char[][] number) {
+        char[][] numberOne = {{' ', ' ', ' ' }, {' ', ' ', '|' }, {' ', ' ', '|' } };
```

Comment:
Maybe you can make this char[][] part of the enum, that way, this reader could be a lot simpler by just getting the ocr representation from the enum.

Reply:
i was modified the enum class and refactor the opticCharacterReader class for read the pattern numbers of this class